### PR TITLE
add apiserver cronjob for longduration cases

### PR DESCRIPTION
**Pull request by dpuniaredhat**
@kuiwang02 @wangke19 Created this PR to add longduration pool for apiserver as discussed in [PR 8822 comment](https://github.com/openshift/openshift-tests-private/pull/8822#discussion_r1177339272 ).

https://github.com/openshift/openshift-tests-private/pull/8822#discussion_r1177339272 

Please help to review.

### CODE REVIEW

**Comment by kuiwang02** [comment]
please fix generated-config failure.
in your local env, you could rebase the latest code from master branch and then `make update`. it should fix your issue.

for other, will review it.

**Comment by dpuniaredhat** [comment]
> please fix generated-config failure. in your local env, you could rebase the latest code from master branch and then `make update`. it should fix your issue.
> 
> for other, will review it.

@kuiwang02 Fixed!

**Comment by kuiwang02** [comment]
iamok from golang's view.

after it is merged, I will make PR to exclude it from long-duration-part1-3.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.10__amd64-nightly.yaml [hunk 1]
```diff
@@ -467,6 +467,18 @@ tests:
     - ref: idp-htpasswd
     - ref: openshift-extended-test-longduration
     workflow: cucushift-installer-rehearse-aws-ipi-private
+- as: aws-ipi-p3-f28-long-duration-apiserver
```
Comment (wangke19) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.10__amd64-nightly.yaml:470]: p3 is lower, p2 is better, [Priority Definition](https://docs.google.com/document/d/1-ODg26Ho2AUKCJ-7kpJizmCrUunjAHteWvlC4yG1PxY/edit#heading=h.2vpiramvcdo1), tested at least once.
f28 means every 28 days, for z-streams, we need to run the case at least once a week. We had better run the test every Friday, check the results on next Monday, f7 is fine.
Cron job: `29 18 * * 5`, run test at 18:29 every Friday.

Comment (dpuniaredhat) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.10__amd64-nightly.yaml]: @wangke19 Updated the suggested changes.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.10__amd64-nightly.yaml [hunk 2]
```diff
@@ -467,6 +467,18 @@ tests:
     - ref: idp-htpasswd
     - ref: openshift-extended-test-longduration
     workflow: cucushift-installer-rehearse-aws-ipi-private
+- as: aws-ipi-p2-f28-long-duration-apiserver
+  cron: 29 18 * * 5
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.10__amd64-nightly.yaml:471]: You can use [script](https://github.com/openshift/release/blob/master/ci-operator/config/openshift/openshift-tests-private/tools/generate-cron-entry.sh) to generate cron entries, which will spread the jobs across the day/month.

Comment (dpuniaredhat) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.10__amd64-nightly.yaml]: Thanks, Fixed!

**Pull request merged**